vendor/golang.org/x/sys/unix/syscall_linux.go

```diff
@@ -342,10 +352,14 @@ func Wait4(pid int, wstatus *WaitStatus, options int, rusage *Rusage) (wpid int,
 	return
 }
 
-func Mkfifo(path string, mode uint32) (err error) {
+func Mkfifo(path string, mode uint32) error {
```

Comment:
__[golint]__
* [golint] exported function Mkfifo should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->